src/plugins/Miner.cpp

```diff
@@ -140,7 +140,8 @@ void DistrubuteMineralWorker(const sc2::Unit* unit_) {
 
 void Miner::OnStep(Builder* builder_) {
     SecureMineralsIncome(builder_);
-    SecureVespeneIncome();
+    if (gAPI->observer().GetGameLoop() % 10 == 0)
```

Comment:
Perhaps the whole plugin should be launched e.g. every 10th frame?
Another interesting question: how does it work with increased frames per turn value and in realtime?

Reply:
Good points. I've refactored this to be part of Plugin (and changed QuarterMaster's frames_to_skip logic to match).